### ISSUE TITLE
missing config.env

### ISSUE THREAD

**Issue by nader-apr** [arashnm80/spot-seek-bot#8]
Missing config.env and need to import load_dotenv from dotenv in variables.py file.

confg.env contains:
SPOT_SEEK_BOT_API = ""
MUSIC_DATABASE_ID = ""
LOG_CHANNEL_ID = ""
SPOTIFY_APP_CLIENT_ID = ""
SPOTIFY_APP_CLIENT_SECRET = ""

**Comment by nader-apr** [comment]
Dotenv doesn't work. The best alternative is Environ.

Put this in the variables.py file:
 
import environ
environ.Env.read_env('config.env')

bot_api = os.environ['SPOT_SEEK_BOT_API']
database_channel = os.environ['MUSIC_DATABASE_ID']
log_channel_id = os.environ['LOG_CHANNEL_ID']
spotify_client_id = os.environ['SPOTIFY_APP_CLIENT_ID']
spotify_client_secret = os.environ['SPOTIFY_APP_CLIENT_SECRET']

**Comment by arashnm80** [comment]
yes the count of env variables is becoming higher. i might do it soon. thanks.
